src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $background-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform $transition-duration, color $transition-duration;
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__tittle {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover &__tittle {
+    color: #34568b;
+  }
+
+  &__code {
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__reviews-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    color: #fff;
+    background-color: $accent-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    border: 1px solid $accent-color;
+    transition: color .3s, background 0.3s;
```

Comment:
You put the `transition` duration into a variable, so it can be used here as well.